lib/matplotlib/backend_tools.py

```diff
@@ -989,12 +989,12 @@ def trigger(self, *args, **kwargs):
                  'help': ToolHelpBase,
                  'copy': ToolCopyToClipboardBase,
                  }
-"""Default tools"""
+# Default tools
```

Comment:
Just delete this comment and the one below, the comment is the same as the variable name :)

Reply:
Done in rebase I just pushed. :+1: